ruby3.2-io-event.yaml

```diff
@@ -1,13 +1,19 @@
 package:
   name: ruby3.2-io-event
-  version: 1.6.5
+  version: 1.7.1
   epoch: 0
   description: An event loop.
   copyright:
     - license: MIT
   dependencies:
     runtime:
-      - ruby-3.2
+      - ruby-${{vars.php-version}}
```

Comment:
This package is not related to php

Reply:
Good spot, bad name / copy/paste of the version-transforms, i'll rename the variable